modules/@apostrophecms/area/ui/apos/components/AposWidgetControls.vue

```diff
@@ -47,6 +47,10 @@ export default {
       default() {
         return {};
       }
+    },
+    item: {
```

Comment:
It doesn't seem this is used. If not, we should remove it here and in `AposAreaWidget.vue` where it's passed in.